{{ cookiecutter.repo_name }}/setup.py

```diff
@@ -1,71 +0,0 @@
-import sys
```

Comment:
@Sparks29032 we need setup.py for c++ packages.  Is it ok that it is deleted here?  I guess we can add it back under a different PR.

Reply:
Yea, should be fine.